stimpacks/aws/login.go

```diff
@@ -108,6 +108,11 @@ func (a *Aws) Login() error {
 			a.log.Debug("Setting {} credentials as default", profileName)
 		}
 		a.aws.SaveProfile(profileName, &profile, defaultProfile, &stimProfile)
+
+		a.log.Info("Saved {} in ~/.aws/credentials", profileName)
+		if defaultProfile {
+			a.log.Info("Set {} credentials as default", profileName)
```

Comment:
Should we remove the `debug` message on line 108? would it be redundant in this case?

Reply:
It seems to me that the debug messages are aspirational, in that they're declaring a wishful desire for an outcome, so that when there's a failure it's likely that it with whatever the last logged declared aspiration was.  Retains its value for that reason.  And the stdout messages only declare what has actually occurred after the fact.

Merely my outsider opinion.

Reply:
Ah, I see they are slightly different - at first glance they looked identical.  You're right.